cray_product_catalog/catalog_update.py

```diff
@@ -129,6 +131,16 @@ def active_field_exists(product_data):
     return any("active" in product_data[version] for version in product_data)
 
 
+def create_config_map(api_instance, name, namespace):
+    """Create new product config map."""
+    new_cm = V1ConfigMap()
+    new_cm.metadata = V1ObjectMeta(name=name)
+    api_instance.create_namespaced_config_map(
+        namespace=namespace, body=new_cm
+    )
+    LOGGER.debug("Created product config map %s/%s", namespace, name)
```

Comment:
To be consistent with the other log messages, change "config map" to "ConfigMap"
```suggestion
    LOGGER.debug("Created product ConfigMap %s/%s", namespace, name)
```